scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -463,3 +522,29 @@ task PlotACNVResults {
         #corresponding output images
     }
 }
+
+#Calling ACNV segments as CNLoH and balanced
+task CNLoHAndSplitsCaller {
+    String entity_id
+    File jar_file
+    File tumor_hets
+    File acnv_seg
```

Comment:
Rename this `acnv_segments`.  By the way, I am being a hypocrite since some of my variable names in the Queue scripts have been less than fully explicit.